src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
-import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../types/Movie';
```

Comment:
Nise solution, i try to do the same but ran into similar error.(i can't see you '../types/Movie') We have index.ts files for it.